packages/adminrouter/extra/dcos-adminrouter-slave.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=Admin Router Agent: A high performance web server and a reverse proxy server
```

Comment:
That sounds sane. But it was my understanding that moving forward, terminology should refer to `agents` instead of `slaves`.